armi/physics/neutronics/settings.py

```diff
@@ -170,6 +184,32 @@ def defineSettings():
             label="Load pad length (cm)",
             description="The length of the load pad. Used to compute average and peak dose.",
         ),
+        setting.Setting(
+            CONF_ACLP_DOSE_LIMIT,
+            default=80.0,
+            label="ALCP dose limit",
+            description="Dose limit in dpa used to position the ACLP",
+        ),
+        setting.Setting(
+            CONF_RESTART_NEUTRONICS,
+            default=False,
+            label="Restart neutronics",
+            description="Restart global flux case using outputs from last time as a guess",
+        ),
+        setting.Setting(
+            CONF_GRID_PLATE_DPA_XS_SET,
+            default="dpa_EBRII_HT9",
+            label="Grid plate DPA XS",
+            description="The cross sections to use for grid plate blocks DPA when computing displacements per atom.",
+            options=CONF_OPT_DPA,
+        ),
+        setting.Setting(
+            CONF_DPA_XS_SET,
+            default="dpa_EBRII_HT9",
+            label="DPA Cross Sections",
+            description="The cross sections to use when computing displacements per atom.",
+            options=CONF_OPT_DPA,
+        ),
```

Comment:
These seem like pretty un-framework-y settings to me...

Reply:
Agreed for sure, but they are required for the framework to run at the moment. (see PR description). This PR fixes broken framework code. Follow-up work should file these somewhere else.